docs/guides/akamai/get-started/parent-child-accounts/index.md

```diff
@@ -0,0 +1,154 @@
+- No access to asset instances created after the creation of the parent/child relationship. The child account parent user needs to ask a child account admin to grant them access to new instances if necessary.
+- The [global permissions](/docs/products/platform/accounts/guides/user-permissions/) set off.
+
+A child account admin can change the child account parent user’s default permissions and expand or restrict its access to the child account, except for billing. If the child account admin provides the child account parent user with full access to the child account, the child account parent user can fully manage the account, including its users.
+
+To switch accounts:
+1. Log in to the [Cloud Manager](https://cloud.linode.com/).
+1. In the top corner, click the name of your account.
+
+    ![Parent account switch](parent_switch.png)
+
+1. Click the **Switch account** button. If you don’t see this button, it means you’re a parent account user with limited access and you need to ask the parent account admin to provide you the access to child accounts or your account is not involved in the parent/child relationship.
+
+    ![Account switch button](switch_account.png)
+
+1. Select an account from the list you want to switch to.
+Now you act as the child account parent user. Note that you can click the **switch back to your account** link to switch back.
+
+    ![switch back option](switch_back.png)
+
+### Delete a parent account
+A parent account can be deleted only if all parent/child relationships are contractually removed from your parent account, meaning you don’t have active contracts with end customers.
+
+Once this condition is met, to close an account:
+
+1. Log in to the [Cloud Manager](https://cloud.linode.com/).
+1. In the main menu, go to **Account**.
+1. In the **Settings** tab, go to the **Close Account** section, and click the **Close Account** button.
+1. Follow the on-screen instructions and click **Close Account**.
+
+## For end customers
+
+### About the feature
+The parent and child accounts feature allows an Akamai partner (Parent account holder) to manage your account (Child account) in Cloud Manager.
+
+You can monitor Akamai partner’s actions on your account as all actions are logged and can be viewed in the Cloud Manager, either through the Events panel or the [Events](https://cloud.linode.com/events) page.
+
+The parent/child relationship doesn’t restrict your account and its users from creating support tickets or signing up for managed services for their accounts.
+
+### Terminology
+In the context of the parent/child relationship, you need to be familiar with the following terms.
+![Parent/child relationship terminolog](parent_child_term.png)
+
+- *Parent account* - a company account of your Akamai partner.
+- *Child account* - your, end customer’s, account.
```

Comment:
Same point I call out above re: bold formatting for lead-ins. Also, the commas aren't necessary:

"- **Child account**. Your end-customer account." (Hyphenated because the term end customer is a multi-word adjective in this context.)